src/App.css

```diff
@@ -22,27 +22,25 @@
   --rewards-text-green: #54AE6F;
   --rewards-background-green: rgba(84, 174, 111, 0.1);
   
-  --rubik: 'Rubik', sans-serif;
-  --open-sans: 'Open Sans', sans-serif;
```

Comment:
We should remove one use of the `--open-sans` variable here: https://github.com/levelkdev/BC-DAPP/blob/master/src/components/common/UserAddress.js#L18 Otherwise looks good!